jenkins/ci.suse.de/templates/cloud-ardana-pipeline-template.yaml

```diff
@@ -39,7 +39,6 @@
             - hosdevelcloud8
             - hosGM8
             - hosGM8+up
-            - stagingcloud9
```

Comment:
I know this looks like a duplication, but if you delete this, then the openstack-ardana job will be missing that option (cloudsource is set to '' for that job)